lib/config/find-prefix.js

```diff
@@ -43,8 +43,14 @@ function findPrefix_ (p, original, cb) {
     // walked up too high or something.
     if (er) return cb(null, original)
 
+    var packageJsonName = 'package.json'
```

Comment:
`npm.config.get` is not available in this context -- is there a better way to determine cli params passed in from this point in the code?